src/components/search/autocomplete.js

```diff
@@ -0,0 +1,107 @@
+import React from "react"
+import axios from "axios"
+import Select, { components } from "react-select"
+import Highlighter from "react-highlight-words"
+
+const getLocationOptions = () => axios.get("/content/locations.json")
+const getZones = () => axios.get("/content/zones.json")
+
+const Autocomplete = ({ setIsZoneDialog, setSelected, selected, setZones }) => {
+  const [locationOptions, setLocationOptions] = React.useState([])
+  const [inputChange, setinputChange] = React.useState()
+
+  const cssBtn = {
+    cursor: "pointer",
+    fontSize: "1.8rem",
+    display: "inline-block",
+    padding: "1rem",
+    background: "var(--cl-black)",
+    color: "var(--cl-white)",
+    fontFamily: "var(--ff-title)",
+    border: "1px solid #000000",
+    borderRadius: " 0 4px 4px 0",
+    display: "flex",
```

Comment:
display ซ่ำสองทีกับบรรทัดที่ 16

Reply:
เอา cssBtn ออกไปอยู่นอก component ก็ดีนะ, there is no point to keep it inside component งะ

Reply:
ตาม design ใหม่ ลบทิ้งไปละ